src/Selenium2Library/keywords/_element.py

```diff
@@ -297,6 +297,21 @@ def get_horizontal_position(self, locator):
             raise AssertionError("Could not determine position for '%s'" % (locator))
         return element.location['x']
 
+    def get_element_size(self, locator):
+        """Returns width and height of element identified by `locator`.
+
+        The element width and height is returned.
+        Fails if a matching element is not found.
+        """
+        element = self._element_find(locator, True, False)
+        if element is None:
+            raise AssertionError("Element with locator '%s' was not found or is not visible" % (locator))
+
+        if element.size['width'] * element.size['height'] == 0:
```

Comment:
I don't believe I contributed to the earlier discussion about zero height/width but I am thinking the same as @aaltat that we don't want to error out on a zero width or height. Here, if the element is found, I would return the width and height and let the user interpret the result. I googled "can element have zero width or height" and it looks like there are legitimate reason for the result to be zero and even possible for either one to be zero and the other non zero thus failing on the
```
if width*height == 0 then error
```
check. Thus I would leave it up to the user to decide what is an error or not. Just my 2 cents...